src/wacks4shop/wack.py

```diff
@@ -98,7 +108,8 @@ def main(db: Wackabase, dry=False):
             Path(DATABASE_DIR).mkdir(parents=True, exist_ok=True)
 
             wackabase = Wackabase(DATABASE_DIR)
-            main(db=wackabase, dry=args.dry)
-            wackabase.write_success()
```

Comment:
we're replacing this write_success with the waterline?

Reply:
`write_success` always wrote out a timestamp to a file location that nothing used. I think it's just a remnant of etsy copy/pasta